### ISSUE TITLE
Error installing archive-tar-minitar

### ISSUE THREAD

**Issue by jordansissel** [halostatue/minitar#23]
I confess that I am not sure what is causing this, but here is the error;

```
% gem install archive-tar-minitar
Fetching: minitar-0.6.gem (100%)
The `minitar` executable is no longer bundled with `minitar`. If you are
expecting this executable, make sure you also install `minitar-cli`.
Successfully installed minitar-0.6
Fetching: archive-tar-minitar-0.6.gem (100%)
ERROR:  Error installing archive-tar-minitar:
        archive-tar-minitar requires minitar-cli (~> 0.6)
```

If I manually install `minitar-cli`, then retry, installation is successful.

This is super weird!

I can reproduce this on ruby 2.2.1 and JRuby 1.7.24.

**Comment by jordansissel** [comment]
I'm going to try and spend some cycles figuring out what is causing this problem. The minitar gemspecs look fine to me. This behavior (by rubygems installer) is confusing.

**Comment by halostatue** [comment]
I have opened a [discussion](http://help.rubygems.org/discussions/problems/25567-gem-install-archive-tar-minitar-is-failing-from-version-06) on the RubyGems help site as I believe that this is an order-of-push issue. When I install the archive-tar-minitar gemfile locally, everything is pulled correctly.

**Comment by jordansissel** [comment]
@halostatue Agreed this looks like a rubygems.org problem. I updated the discussion you started. Summary: The rubygems.org dependency api does not show `minitar-cli` as a dependency even when the .gem file does.

**Comment by halostatue** [comment]
I’ll probably be able to fix it with a 0.6.1 release (released in the right order this time…), but given that there’s an issue with 0.6 and gzip streams/files that no one can seem to tell me the best way to repro in a Vagrant box…, I don’t want to update archive-tar-minitar until I have that fixed.
